f/nametag/functions/fn_nametagGetData.sqf

```diff
@@ -245,7 +258,10 @@ private _zoom = 1;
 					//	For units in the same group as the player, set their color according to color team.
 					if _sameGroup then 
 					{
-						_team = assignedTeam _x;
+						_team = assignedTeam _x; //Note: may be nil
+						if( isNil "_team") then {
+							_team = "";
+						};
 						_nameColor = switch _team do 
```

Comment:
This and the line before would throw a script error ingame because of uninitialized variables.